src/index.ts

```diff
@@ -55,12 +75,19 @@ export default (options: Options) => (
       protocol: req.protocol.toLowerCase() as HttpProtocol,
       body: typeof req.body === "string" ? req.body : JSON.stringify(req.body)
     });
+
     const response = HttpResponseBuilder.from({
       statusCode: res.statusCode,
       body,
       headers: res.getHeaders() as { string: string | string[] }
     });
-    options.writer(JSON.stringify({ request, response }));
+
+    const exchange: HttpExchange = { request, response };
+
+    transports.forEach(async transport => {
+      await transport(exchange);
```

Comment:
Maybe a bad idea to await here as it blocks the response but we can fix that later, I think.